go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/mt-sre/devkube
+
+go 1.17
```

Comment:
Go version used inside of Jenkins is 1.15, and the image we use in AMO is not yet 1.17 friendly.

Stuck at 1.16 still -> https://catalog.redhat.com/software/containers/rhel8/go-toolset/5b9c810add19c70b45cbd666

I assume we might want to run CI inside a docker container sandbox to avoid all the issues we have faced in AMO. https://github.com/mt-sre/addon-metadata-operator/blob/master/Dockerfile.ci

Reply:
I will downgrade the module to 1.16, but yes especially on Jenkins I would rather do everything in a container.
Not sure if there is even a 1.17 golang image on OpenShift CI yet.